sqldelight-idea-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/SqlDelightProjectComponent.kt

```diff
@@ -19,6 +19,8 @@ import org.jetbrains.kotlin.idea.util.projectStructure.getModule
 class SqlDelightProjectComponent(
   private val project: Project
 ) : ProjectComponent, VirtualFileListener {
+  private var removeListeners: (() -> Unit)? = null
```

Comment:
Pluralization is kinda weird. Expected a list. How about `closeListener`?

Reply:
cool wit me